source/opt/optimizer.cpp

```diff
@@ -494,6 +494,8 @@ bool Optimizer::RegisterPassFromFlag(const std::string& flag) {
     RegisterPass(CreateWrapOpKillPass());
   } else if (pass_name == "amd-ext-to-khr") {
     RegisterPass(CreateAmdExtToKhrPass());
+  } else if (pass_name == "interp-fixup") {
```

Comment:
Again, please use the full word "Interpolation".

Reply:
OK